src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+ *     <li></li>
+ * </ul>
+ *
+ * <h3>Output (currently)</h3>
+ * <ul>
+ *     <li>prints to screen a summary table containing summary on the TP/FP/FN/FFN/FFP numbers.</li>
+ *     <li>writes a BED-4 file containing: TP, FP, FILTERED_TRUE_NEGATIVE (record correctly filtered), and FILTERED_FALSE_NEGATIVE</li>
+ * </ul>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *   gatk EvaluateConcordanceOnNonGenotypedSVVCFs \
+ *     -T truth.vcf \
+ *     -E evalCallSet.vcf \
+ *     -O outputPrefix
+ * </pre>
+ */
+@DocumentedFeature
+@BetaFeature
+@CommandLineProgramProperties(
+        oneLineSummary = "(Internal) Evaluates concordance between given call set and a single unique truth set for non-genotyped SV vcf",
+        summary = "This tool is used in development and should not be of interest to most researchers. " +
+                "It is a prototype of evaluating non-genotyped structural variant calls against a single truth, " +
+                "and has been under active developments. " +
+                "This tool pads the interval around the interval bounded by POS and END as given in each record, " +
+                "by an amount specified in the argument \"padding\" to look for matching truth record, then " +
+                "tests if the haplotypes by the truth and the evaluation calls are concordant. ",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class)
+public final class EvaluateConcordanceOnNonGenotypedSVVCFs extends AbstractIntervalTreeBasedConcordanceWalker {
+
+    @Argument(doc = "file containing non-canonical chromosome names (e.g chrUn_KI270588v1) in the reference; " +
+            "all chromosome will be considered canonical if not provided",
+            shortName = "alt-tigs",
+            fullName = "non-canonical-contig-names-file",
+            optional = true)
+    private String nonCanonicalChromosomeNamesFile;
+
+    @Advanced
+    @Argument(doc = "indicate evaluation is being performed on non-human calls",
+            shortName = "nonhs",
+            fullName = "non-human",
+            optional = true)
+    private Boolean nonHumanVCF = false;
+
+    @Advanced
+    @Argument(doc = "string value for chromosome Y; takes value of \"Y\" or \"chrY\", or when left unprovided, " +
+            "evaluation is skipped on chromosome Y; applicable only when \"non-human\" is false",
+            shortName = "chry",
+            fullName = "chr-y-string",
+            optional = true)
+    private String chrYString;
+
+    /**
+     * A fixed amount of padding for each VCF record from the eval call set.
+     * Each individual record might be padded (more or less bases) as judged appropriate by the tool itself.
+     */
+    @Argument(doc = "Padding around variants for overlapping",
+            fullName = "padding",
+            optional = true)
+    private Integer fixedPadding = 50;
```

Comment:
Why ```Integer``` and not ```int```.

Reply:
does it make a difference? I'm just curious. Is it for `null`?